meson.build

```diff
@@ -0,0 +1,73 @@
+project('eos-event-recorder-daemon',
+        'c',
+        license: 'GPL-2.0-or-later',
+        version: '0.0.0',
+        default_options: ['c_std=c11'],
+        meson_version: '>= 0.52.0',
+)
+
+cc = meson.get_compiler('c')
+
+eosmetrics_dep = dependency('eosmetrics-0', version: '>= 0.2')
+polkit_gobject_dep = dependency('polkit-gobject-1')
+
+emer_shared_required_modules = [
+    dependency('gio-unix-2.0', version: '>= 2.46'),
+    dependency('gobject-2.0', version: '>= 2.58'),
+    dependency('sqlite3'),
+    dependency('uuid'),
+]
+emer_required_modules = emer_shared_required_modules + [
+    eosmetrics_dep,
+    cc.find_library('m', required: false),
+    dependency('glib-2.0', version: '>= 2.63.1'),
```

Comment:
Being pedantic, `glib-2.0` is almost certainly a shared required module, not a private one